documentation/source/license/main.rst

```diff
@@ -1,6 +1,6 @@
-*******
-License
-*******
+***********
+MIT license
```

Comment:
Would not change it. As it is the topic about the license, i would not be more specific here.

Reply:
Done.